model/gen-ai/spans.yaml

```
@@ -157,30 +164,38 @@ groups:
       The `gen_ai.operation.name` SHOULD be `create_agent`.
       The **span name** SHOULD be `create_agent {gen_ai.agent.name}`.
       Semantic conventions for individual GenAI systems and frameworks MAY specify different span name format.
-    extends: trace.gen_ai.client.common_attributes
+    extends: trace.gen_ai.client.common_request_attributes
     attributes:
       - ref: gen_ai.system
         requirement_level: required
-      - ref: gen_ai.agent.id
-        requirement_level:
-          conditionally_required: if applicable.
       - ref: gen_ai.agent.name
         requirement_level:
           conditionally_required: If provided by the application.
       - ref: gen_ai.agent.description
         requirement_level:
           conditionally_required: If provided by the application.
-      - ref: gen_ai.request.model
+
+  - id: span.gen_ai.client.invoke_agent
+    type: span
+    stability: experimental
```

we'll start requiring span kind soon, let's explicitly specify it here
```suggestion
    stability: experimental
    span_kind: client
```

it'd also force you to change the group id to `span.gen_ai.invoke_agent.client`

---

Done